src/sessionManager.ts

```diff
@@ -44,6 +44,7 @@ export default function SessionManager(
                 self.endSession();
                 self.startNewSession();
             } else {
+                // QUESTION: Should we check for current user if we don't have persistence?
```

Comment:
I would say no.  But the no persistence state is not a realistic use case and more an intermediate state to get us to plug and play persistence types, if that changes what you were thinking about doing here.